docs/fragments/imagemin.md

```diff
@@ -0,0 +1,6 @@
+:::caution
+The `imagemin` library is [officially unmaintained][imagemin is unmaintained].
```

Comment:
Curious what the purpose of this file is and where the "fragments" folder fits into the rest of the docs structures...?

Reply:
Fragments is in anticipation of the Tauri + <Bundler/Framework> guides. 
Those docs will share a lot of content and extracting common parts into shared files will make it easier to keep them in sync